init.d/root.in

```diff
@@ -49,8 +49,6 @@ start()
 	local mountpoint
 	for mountpoint in $(fstabinfo); do
```

Comment:
I suppose it could be simplified to:
```sh
	for mountpoint in $(fstabinfo); do
		mountinfo -q "${mountpoint}" && \
			fstabinfo --remount "${mountpoint}"
	done
```

Reply:
Good point, updated accordingly.

@williamh did you have a chance to think about this yet? If you need more information let me know :)

Reply:
This check is actually necessary as otherwise it will also attempt to remount swap.